go/vt/vtorc/db/generate_base.go

```diff
@@ -23,6 +23,7 @@ DROP TABLE IF EXISTS database_instance
 `,
 	`
 CREATE TABLE database_instance (
```

Comment:
If I'm reading this correctly, this is the only table where we still retain (hostname, port) because we do need that information. It's deleted from all other tables and replaced with alias, correct?

Reply:
Yes, we need the information because at the MySQL level we only the `source_host` and `source_port` and not the `source_alias`, so in order to know which tablet a certain MySQL instance is replicating from, we need to know the `host` and `port` of the tablet.